templates/containers/chiseled/index.html

```diff
@@ -508,8 +508,7 @@ <h3 class="p-text--small-caps">Documentation</h3>
           <div class="col-4 col-medium-3">
             <ul class="p-list--divided">
               <li class="p-list__item">
-                <a href="https://canonical-rockcraft.readthedocs-hosted.com/en/latest/explanation/chisel/">Using chisel in
-                Rockcraft</a>
+                <a href="https://documentation.ubuntu.com/rockcraft/en/latest/how-to/rocks/chisel-existing-rock.html">Using chisel in Rockcraft</a>
               </li>
```

Comment:
Won't let me comment directly on the line, but the link for "Install packages slices in a rock" is returning 404